lightning/src/ln/router.rs

```diff
@@ -427,6 +452,7 @@ impl RoutingMessageHandler for Router {
 					},
 					None => {},
 				}
+				log_trace!(self, "Announced node {}", msg.contents.node_id);
```

Comment:
This is a super confusing wording. Instead of "Announced" (here and below), can we use like "Received valid announcement for..."? Its probably also worth including the should_relay field (here and further down) to indicate if we intend to relay it onwards.